resources/xlf/de/sql.de.xlf

```diff
@@ -692,6 +692,10 @@
       <trans-unit id="notebook.sqlStopOnError">
         <source xml:lang="en">SQL kernel: stop Notebook execution when error occurs in a cell.</source>
         <target state="translated">SQL-Kernel: Notebook-Ausführung bei Fehler in Zelle beenden</target>
+      </trans-unit>
+			<trans-unit id="notebook.trustedNotebooks">
```

Comment:
1. You have a mix of tabs/spaces. I'd suggest turning on whitespace visualization to catch these kinds of things before submit
2. Why are you adding these strings manually? Where did you get them? Usually the loc team handles updating these - we can just add the localize calls in the code and it will be turned into entries in the xlf files automatically